contact.html

```diff
@@ -1,36 +1,68 @@
 <!DOCTYPE html>
-<html>
+<html lang="en" class="has-navbar-fixed-top">
```

Comment:
Is there a reason why `has-navbar-fixed-top` is required here? It seems a bit strange to have it in the `html` tag; I also noticed this in `index.html` but not in the other HTML files. I haven't used Bulma before though, so I might just be missing something.

Reply:
`has-navbar-fixed-top` makes it so that Bulma handles the actual site content not overlapping or appearning behind a fixed navbar (see docs section on it ![here](https://bulma.io/documentation/components/navbar/#fixed-navbar)). The tag has to be on either `html` or `body` so if you think it would make more sense on `body` instead I'm happy to move it there.

It should be on the other pages too but I guess I must have just done it manually on those ones, it might be a good idea to create a follow up PR to make them all consistent.

Reply:
Ah I see, thanks! I think it would make a bit more sense to have it on the body tag, if that's alright.

Reply:
Sounds good 👍 , I'll add it to this page in this PR but I think I'll move the other pages over in the aforementioned follow-up if that's alright with you?

Reply:
Sounds good!